R/metadata.R

```diff
@@ -3,6 +3,16 @@ renv_metadata_version <- function() {
   `_renv_metadata`$sha %||% `_renv_metadata`$version
 }
 
+renv_metadata_version_friendly <- function(metadata = `_renv_metadata`) {
+  ver <- metadata$version
```

Comment:
Is this always the package version, or might it sometimes be the sha? See: https://github.com/rstudio/packrat/pull/713/files?w=1#diff-485d0d5662ab5285bfe374325ff46ad3bff5a6ef9ea4a38914744270e9a567f5R19

Maybe we need to record both the package version, as well as the commit hash, here?

https://github.com/rstudio/renv/blob/4c88614a182d4dc1e1e9e5a309dc1bed01c5714e/inst/resources/vendor/renv.R#L10-L14

Reply:
Yeah, I think so.

Reply:
I took a slightly different approach to ensure that the vendored metadata always stays in sync with the regular metadata

Reply:
Good idea!